Example/ExampleApp/Responder/ResponderViewController.swift

```diff
@@ -150,8 +150,8 @@ extension ResponderViewController: SessionViewControllerDelegate {
         print("[RESPONDER] Approving session...")
         let proposal = currentProposal!
         currentProposal = nil
-        let accounts = proposal.permissions.blockchains.map {$0+":\(account)"}
-        client.approve(proposal: proposal, accounts: Set(accounts))
+        let accounts = Set(proposal.permissions.blockchains.compactMap { Account($0+":\(account)") })
```

Comment:
can we add another `convenience init` to avoid passing the `:` , It looks easy to forget